examples/cube.rs

```diff
@@ -148,7 +148,7 @@ void main() {
             Vertex::new([1, -1, -1], [0, 1]),
         ];
 
-        #[cfg_attr(rustfmt, rustfmt_skip)]
+        #[rustfmt::skip]
```

Comment:
Stabilization of tool attributes is probably tied to a newer version of Rust. Let me know if it is a desirable change.